master/cluster.go

```diff
@@ -883,6 +883,9 @@ func (c *Cluster) batchCreatePreLoadDataPartition(vol *Vol, preload *DataPartiti
 }
 
 func (c *Cluster) batchCreateDataPartition(vol *Vol, reqCount int) (err error) {
+	if _, err = vol.needCreateDataPartition(); err != nil {
+		return
```

Comment:
It is recommended to add a warn alarm log here and print out err

Reply:
Thanks for your review.